src/main/java/org/pentaho/mongo/wrapper/DefaultMongoClientFactory.java

```diff
@@ -17,10 +17,7 @@
 
 package org.pentaho.mongo.wrapper;
 
-import com.mongodb.MongoClient;
-import com.mongodb.MongoClientOptions;
-import com.mongodb.MongoCredential;
-import com.mongodb.ServerAddress;
+import com.mongodb.*;
```

Comment:
Change this section back; our style guidelines specify that each import must be included separately, even if we're importing the entire package.

Reply:
Changed